docs/home/whats-new-2-18.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Version 2.18
+summary: This article describes new features and functionality in the version.
+authors:
+    - Jamie Weider
+date: 2024-June-14
+---
+
+## Release Content - June 30, 2024
+
+* [Deprecation notifications](#deprecation-notifications)
+* [Breaking changes](#breaking-changes)
+
+### Researcher
+
+#### Jobs, Workloads, and Workspaces
+
+* <!-- TODO ADD LINK TO DOC Run-14732/Run-14733 Add backoff limit to workspace & standard training -->Added backoff limit functionality to Training and Workspace workloads. The backoff limit is the maximum number of retry attempts for failed workloads. After reaching the limit, the workload's status will change to `Failed`.
+
+* <!-- RUN-16830/RUN-16831 - Graphs & special metrics for inference -->Added new graphs for *Inference* workloads. The new graphs provide more information for *Inference* workloads to help analyze performance of the workloads. For more information, see [Workloads View](../admin/workloads/README.md#workloads-view).
+
+* <!-- RUN-18944/RUN-18945 Changing "Auto-deletion" default and presentation of the default value in the UI -->Updated *Auto-deletion time* default value. Previously the default was **90 days** now the default is **30 days**.
+
+* <!-- TODO better explanation RUN-16917/RUN-19363 Expose secrets in workload submission -->Added new data sources to workload to include project secrets.
+
+* <!-- TODO add link to doc when ready - get approval for text RUN-16805/RUN-17416 - Provide latency-based metric for autoscaling for requests -->Added latency metric for autoscaling. This feature is used to set a target threshold for the response time of requests. This will adjust the number of applications to keep the response time below that threshold. For more information, see [Auto scaling]().
+
+* <!-- TODO Add docs for models explanation. RUN-16872/RUN-18526 Separating ChatUi from model in favor of coherent autoscaling -->Improved autoscaling for ChatUi models. Run:ai has improved autoscaling performance with ChatI models by adding them to *Environments*. ChatUi is an addition to inference workloads and is not mandatory for all types of workloads. For more information, see [Models]().
```

Comment:
The impact of the change is performance improvement of autoscaling? Interesting.... @liohill , is this really the case?